robotbase/src/main/java/frc/robot/commands/LEDs/LEDsSetColor.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.commands.LEDs;
+
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Robot;
+
+public class LEDsSetColor extends Command {
+
+  Color m_color;
+
+  public LEDsSetColor(Color color) {
+    addRequirements(Robot.LEDs);
+    m_color = color;
+  }
+
+  @Override
+  public void initialize() {
```

Comment:
In commands that return instantly we typically do the action in the end() method.